### PR TITLE
date and time new functionality added.

### PR DESCRIPTION
**Pull request by HARSHJAIN47**
date and feature added user can access date and time with these params :-

`dateTimeToggle: {
              date: true,
              time: true,
            },`

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { uglify } from 'rollup-plugin-uglify';
 const extensions = ['.ts', '.tsx'];
 
 const indexConfig = {
+  context: 'this',
```
Comment (HenryHengZJ) [inline rollup.config.js:18]: wondering do we need this? how is this being used

**Review by HenryHengZJ — APPROVED** [review]

File: rollup.config.js (hunk 1 above)
Comment (HARSHJAIN47) [reply rollup.config.js:18]: @HenryHengZJ while installing i got error of "this" so thats why i added here.

**Review by HenryHengZJ — APPROVED** [review]
thanks @HARSHJAIN47 !

**Comment by bendadaniel** [comment]
@HARSHJAIN47 Hi, can you please explain how does it works? Thank you

**Pull request merged**